tests/runtime/probe

```diff
@@ -278,6 +278,18 @@ REQUIRES bpftool
 REQUIRES_FEATURE uprobe_multi
 BEFORE ./testprogs/uprobe_test
 
+NAME kprobe_session_progs
+RUN {{BPFTRACE}} --unsafe -e 'kprobe:ksys_* { printf("progs: "); system("bpftool prog | grep kprobe | grep ksys_ | wc -l"); exit(); } kretprobe:ksys_* { exit(); }'
+EXPECT progs: 1
+REQUIRES bpftool
+REQUIRES_FEATURE kprobe_session
+
+NAME kprobe_session_links
+RUN {{BPFTRACE}} --unsafe -e 'kprobe:ksys_* { printf("link: "); system("bpftool link | grep kprobe_multi | wc -l"); exit(); } kretprobe:ksys_* { exit(); }'
```

Comment:
also why they use of the system function?

Reply:
Because that's currently the only way to run bpftool and check that the amount of BPF progs/links is as expected. I'm considering adding a new kind of check, `EXPECT_CMD`, which will execute a command and check the return code but that will be done in a different PR.

Reply:
>  check that the amount of BPF progs/links is as expected

Ah, that part totally went over my head. Maybe a comment wouldn't be a bad idea - for us slow folk 😄

Reply:
Sure, I added the comment.